features/FEATURE_BLE/ble/generic/GenericSecurityManager.h

```diff
@@ -443,6 +468,14 @@ class GenericSecurityManager : public SecurityManager,
     bool _master_sends_keys;
     bool _public_keys_generated;
 
+    /** There is always only one OOB data set stored at a time (for now) */
```

Comment:
there will always be only one set of data stored, otherwise oob exchange would be insecure

Reply:
What if you have pairing on multiple connections ? 
It's far stretched but I'm asking.